src/docs/admin/references/ServiceNowReference.adoc

```diff
@@ -0,0 +1,560 @@
+. Calling the submitFromCatalog function with the inputs as parameters
+. Writing the result in the outputs variable
+
+Finally, we define the outputs of the action with the same pills system, binding them to the step outputs.
+
+.Outputs of the Submit From Catalog Action
+image::servicenow-action-outputs.png[align="center"]
+
+The concept is the same for all the application's Actions.
+
+===== Flows
+
+To put it simply a flow is an automated process consisting of a sequence of actions and a trigger.
+Flow building takes place in the Flow Designer application which requires a paid subscription to the Integration Hub application.
+
+A Flow consists of the content types:
+
+- Subflows: A Subflow is an automated process consisting of a sequence of reusable actions and specific data inputs that allow the process to be started from a flow, subflow, or script.
+- Actions: An Action is a reusable operation that enables process analysts to automate Now Platform features without having to write code
+- Core Actions: A ServiceNow Core Action is a ServiceNow-provided action available to any flow that cannot be viewed or edited from the Action Designer design environment.
+For example, the Ask for Approval action
+
+A trigger identifies what causes the flow to execute. +
+
+Flows are triggered by:
+
+- Record creation and/or update
+- Date
+- Service Catalog request
+- Inbound email
+- Service Level Agreements
+- MetricBase (requires the MetricBase plugin)
+
+[NOTE]
+To access Flows, a user must have the flow_designer or admin roles.
+
+The ProActive application come with Flows that can:
+
+- Submit a job from the catalog asynchronously
+- Restart in error tasks
+
+Below is the "Submit job from catalog async" Flow
+
+image::servicenow-flow.png[align="center"]
+
+This Flow submits a job to ProActive's Scheduler in parallel, thus not blocking the main Flow thread.
+
+In the Right panel named Data, you will find the data flow of the Flow. It offers a quick view of the Flow variables, accessible by all actions
+and the inputs for each Action. Fo convenience, it has been cut in half in this screenshot.
+
+image::servicenow-data-flow-flow-designer.png[align="center"]
+
+The flow needs a trigger definition, and it will ready to run.
+
+===== Workflows
+
+The second way to automate complex processes is using Workflows that are built in the Workflow Editor.
+A Workflow is the native way to do automation in ServiceNow. Workflow editor is intended for a more technical audience.
+
+The goal and concepts of Workflows are the same as Flows only with a different interface and terminology.
+
+Like Actions for Flows, a Workflow is a suite of Activities used for automating the processes.
+Activities are the workflow blocks that carry out various tasks like sending emails, obtaining approvals, running scripts, and configuring field values on the records.
+
+Every workflow starts with a Begin activity and ends with the End activity.
```

Comment:
with a Begin activity and ends with an End activity

OR

with the Begin activity and ends with the End activity.